layouts/_default/baseof.html

```diff
@@ -1,40 +1,19 @@
 <!DOCTYPE html>
 <html lang="{{ .Site.Language.Lang }}" data-type="{{.Type}}" data-relpermalink="{{.RelPermalink}}" style="opacity:0" class="{{ if or $.Site.Params.announcement_banner.link $.Site.Params.announcement_banner.desktop_message }}banner{{ end }}">
 <head>
+    <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-us.js"></script>
     <script>
-      // Init browser logs when the lib is loaded.
-      window.blAsyncInit = function () {
-        // Set your Public API key
-        DD_LOGS.init({
-          publicApiKey: 'pub749f1865c95056099fe5280945238f49',
-          forwardErrorsToLogs: true,
-        });
-        DD_LOGS.addLoggerGlobalContext('env', '{{ $.Site.Params.environment }}');
-        DD_LOGS.addLoggerGlobalContext('service', 'docs');
-        DD_LOGS.addLoggerGlobalContext('host', window.location.host);
-      };
-
-      // Load the Browser logs lib asynchronously if it's not already loaded.
-      (function(d){
-         let js, id = 'browser-logs-js', ref = d.getElementsByTagName('script')[0];
-         if (d.getElementById(id)) {return;}
-         js = d.createElement('script'); js.id = id; js.async = true;
-         js.src = "//www.datadoghq-browser-agent.com/datadog-logs-us.js";
-         const ddScript = ref.parentNode.insertBefore(js, ref);
-         ddScript.onload = function(){window.blAsyncInit();}
-       }(document));
-    </script>
+      // Set your Public API key
+      DD_LOGS.init({
+        publicApiKey: 'pubf839012dcb041ba2b4478770b9e96d11',
+        forwardErrorsToLogs: false,
+      });
```

Comment:
I'm not sure of the impact but this script will be render blocking now. do we want an `async` or `defer` on the script tag? also we don't want to capture errors anymore `forwardErrorsToLogs: false` ?